packages/2d/src/lib/components/TxtLeaf.ts

```diff
@@ -139,7 +139,7 @@ export class TxtLeaf extends Shape {
       -size.height / 2,
       bbox.width,
       bbox.height,
-    ).expand(lineWidth * miterLimitCoefficient);
+    ).expand([0, this.fontSize() * 0.5 * this.absoluteScale().x]).expand(lineWidth * miterLimitCoefficient);
```

Comment:
To quote the docs of `getCacheBBox`:
> The returned bounding box should be in local space.

Is there a reason to multiply the font size by `absoluteScale` here?

Reply:
There is not, I just didn't notice that it was in local space. Will fix.